test/refactoring/rename/Outputs/local/casebind_2.swift.expected

```diff
@@ -21,7 +21,7 @@ func test3(arg: Int?) {
   case let .some(x) where x == 0:
     print(x)
   case let .some(xRenamed) where xRenamed == 1,
-       let .some(x) where xRenamed == 2: // FIXME: This 'x' in '.some(x)' isn't properly renamed in 'casebind_2' case.
+       let .some(x) where x == 2: // FIXME: This 'x' in '.some(x)' isn't properly renamed in 'casebind_2' case.
```

Comment:
Can you look into what's going on here?

Reply:
Sure, can do! But this is a pre-existing refactoring bug. The user would expect all the `x`s to become `xRenamed` but the `.some(x)` in the second pattern is a different decl and so it doesn't change. This PR makes it so that the `x` in the second where clause is the same decl as that second  `.some(x)`, so it doesn't change either.

So more 'correct', but still not the user expectation.

Reply:
Oh ok, makes sense then. I was just wondering why the behavior changes.